### ISSUE TITLE
Seems to require absolute path in remote

### ISSUE THREAD

**Issue by anamba** [mislav/git-deploy#57]
Relative paths in the remote specification (i.e. starting from `$HOME`) should be left unharmed. Currently, `git deploy setup` appears to add a '/' before the path, which will break any relative path.

**Comment by stefanotorresi** [comment]
I'm also experiencing this problem. Having a remote like `user@host:directory/repo` should mean `repo` is deployed into `$HOME/directory`, not `/directory`.

**Comment by mislav** [comment]
I agree that relative paths should not be transformed into absolute. Does anyone want to take a stab at a pull request?
